src/main/java/uk/gov/hmcts/reform/iahomeofficeintegrationapi/domain/entities/ccd/NationalityFieldValue.java

```diff
@@ -0,0 +1,17 @@
+package uk.gov.hmcts.reform.iahomeofficeintegrationapi.domain.entities.ccd;
+
+public class NationalityFieldValue {
```

Comment:
This is already implemented in RIA-3628, which will be merged to master. Rebase on master should cover this. No change required

Reply:
same as above